mephisto/client/review/review_server.py

```diff
@@ -85,10 +89,62 @@ def consume_data():
             ready_for_next.wait()
         finished = True
 
+    # returns all data or page of all data given a limit where pages are 1 indexed
```

Comment:
You may want to write this comment as a [Python docstring](https://www.programiz.com/python-programming/docstrings) instead, such that IDEs can pick up and surface documentation for it.